website/content/api-docs/system/managed-keys.mdx

```diff
@@ -0,0 +1,194 @@
+---
+layout: api
+page_title: /sys/managed-keys - HTTP API
+description: The `/sys/managed-keys` endpoint is used manage the managed keys in Vault.
+---
+
+# `/sys/managed-keys`
+
+The `/sys/managed-keys` endpoint is used manage the managed keys in Vault. Managed keys are keys managed outside
+of Vault's storage and can be leveraged to perform various operations within Vault.
+
+## List Managed keys.
+
+This endpoints lists all the managed keys within the namespace.
+
+| Method | Path                      |
+|:-------|:--------------------------|
+| `LIST` | `/sys/managed-keys/:type` |
+
+### Parameters
+- `type` `(string: <required>)` – The backend type that will be leveraged for the managed key.
+  Supported options are `pkcs11`.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request LIST \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/sys/managed-keys/pkcs11
+```
+
+### Sample Response
+
+```json
+{
+  "keys": [
+    "hsm-key1",
+    "hsm-key2"
+  ]
+}
+```
+
+## Create/Update Managed Key
+
+An endpoint that will create or update a managed key within a given namespace. The :type refers to the backend type
+that the key is supposed to use such as pkcs11. The :name argument is unique name within all managed key types within
+the namespace.
+
+| Method | Path                            |
+|:-------|:--------------------------------|
+| `POST` | `/sys/managed-keys/:type/:name` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request POST \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/sys/managed-keys/pkcs11/hsm-key1
+```
+
+### Generic Parameters
+- `name` `(string: <required>)` - A unique lowercase name that serves as identifying the key. Must be
+  unique throughout the namespace.
+
+- `type` `(string: <required>)` – The backend type that will be leveraged for the managed key.
+  Supported options are `pkcs11`.
+
+- `allow_generate_key` `(string: "false")` - If no existing key can be found in the referenced backend, instructs
+  Vault to generate a key within the backend.
+
+- `allow_replace_key` `(string: "false")` - Controls the ability for Vault to replace through generation or importing
+  a key into the configured backend even if a key is present, if set to false those operations are forbidden
+  if a key exists.
+
+- `allow_store_key` `(string: "false")` - Controls the ability for Vault to generate or import a key within the
+  configured backend, if "false" those operations will be forbidden.
+
+- `any_mount` `(string: "false")` - Allow usage from any mount point within the namespace if "true". If "false"
+  specific mount points will need to be tuned setting `allowed_managed_keys` with the name value of the key
```

Comment:
Maybe "be tuned to set..."?

Reply:
Updated to 

`Allow usage from any mount point within the namespace if "true". If "false" specific mount points will need their allowed_managed_keys parameter to be updated with the key name to grant usage.`